services/brig/src/Brig/Team/API.hs

```diff
@@ -154,6 +156,19 @@ routesPublic = do
     Doc.response 404 "No pending invitations exists." Doc.end
     Doc.response 409 "Multiple conflicting invitations to different teams exists." Doc.end
 
+  get "/teams/invitations/by-email" (continue getInvitationByEmailH) $
+    accept "application" "json"
+      .&. query "email"
+
+  -- TODO: check if this correct
+  document "GET" "getInvitationByEmail" $ do
+    Doc.summary "Get a pending invitation for a given an email address."
+    Doc.parameter Doc.Query "email" Doc.bytes' $
+      Doc.description "Email address"
+    Doc.returns (Doc.ref Public.modelTeamInvitation)
+    Doc.response 404 "No pending invitations exists." Doc.end
```

Comment:
```suggestion
    Doc.response 200 "Pending invitation exists." Doc.end
    Doc.response 404 "No pending invitations exists." Doc.end
```

Looks right to me!  I tried to make a suggestion for the entire block to remove the TODO as well, but github didn't figure out a good diff, so I'm making two separate suggestions.